MyBlog.Services/Areas/Auth/AuthService.cs

```diff
@@ -7,18 +7,20 @@
 using MyBlog.Service.Areas.Users;
 using MyBlog.Service.Areas.Users.AutoMapper.Dto;
 using MyBlog.Service.Exception;
+using MyBlog.Service.Helpers.EncryptDecrypt;
 
 namespace MyBlog.Service.Areas.Auth;
 
 public class AuthService : IAuthService
 {
     private readonly IUserService _userService;
     private readonly IConfiguration _config;
-    
-    public AuthService(IUserService userService, IConfiguration config)
+    private readonly IEncryptDecryptManager _encryptDecrypt;
+    public AuthService(IUserService userService, IConfiguration config, IEncryptDecryptManager encryptDecrypt)
```

Comment:
encrypt - decrypt - it is good to name methods like this, but
think about, how you should call your variable, it must be simple and easy to understand to other developer what is this object.